backend/clickhouse/migrations/000118_write_event_identifier.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE MATERIALIZED VIEW IF NOT EXISTS event_identifier_mv TO event_key_values (
+    `ProjectId` Int32,
+    `Key` LowCardinality(String),
+    `Day` DateTime,
+    `Value` String,
+    `Count` UInt64
+) AS
+SELECT ProjectID as ProjectId,
+    'identifier' AS Key,
+    toStartOfDay(CreatedAt) AS Day,
+    Identifier AS Value,
+    count() AS Count
+FROM sessions
```

Comment:
for any top-level session properties, could we just look up the values from the `fields` table instead? these MVs are doing some duplicate work writing these to both locations